package.json

```diff
@@ -1,14 +1,19 @@
 {
   "name": "eyes.it",
-  "version": "2.0.0",
+  "version": "2.0.1",
   "description": "",
-  "main": "index.js",
+  "main": "dist/src/index.js",
+  "files": [
+    "dist"
+  ],
   "publishConfig": {
     "registry": "https://registry.npmjs.org/"
   },
   "scripts": {
-    "build": ":",
-    "test": "mocha --require eyes.selenium test/index.test.js",
+    "precommit": "lint-staged",
+    "pretest": "yoshi build",
+    "test": "yoshi test --protractor",
+    "build": "yoshi build",
```

Comment:
this line is redundant

Reply:
Actually. Without real e2e tests (which we don't have yet).
We don't have to build before we test.
So We can keep this `"build": "yoshi build"` and remove this `"pretest": "yoshi build"`,
right?
@shlomitc

Reply:
keep the pretest, remove the build